packages/manager/apps/web/package.json

```diff
@@ -100,6 +99,7 @@
     "filesize": "^3.5.10",
     "flatpickr": "^4.5.7",
     "font-awesome": "^4.5.0",
+    "hubspot-messenger": "HubSpot/messenger#^1.4.1",
```

Comment:
Sure this is still used ?

Reply:
https://github.hubspot.com/messenger/docs/welcome/ < I've never seen that

Reply:
yes you're right it's still used by ng-ovh-toaster but doesn't seem to be used in web anymore i'm going to remove this dep